tests/certification/pubsub/rabbitmq/rabbitmq_test.go

```diff
@@ -707,6 +711,109 @@ func TestRabbitMQExtAuth(t *testing.T) {
 		Run()
 }
 
+func TestRabbitMQPriority(t *testing.T) {
+	rand.Seed(time.Now().UTC().UnixNano())
+	log := logger.NewLogger("dapr.components")
+	// log.SetOutputLevel(logger.DebugLevel)
+
+	fullMessages := watcher.NewUnordered()
+	// Application logic that tracks messages from a topic.
+	application := func(pubsubName, topic string, w *watcher.Watcher) app.SetupFn {
+		return func(ctx flow.Context, s common.Service) (err error) {
+			// Simulate periodic errors.
+			sim := simulate.PeriodicError(ctx, errFrequency)
+			return s.AddTopicEventHandler(&common.Subscription{
+				PubsubName: pubsubName,
+				Topic:      topic,
+				Route:      "/" + topic,
+				Metadata:   map[string]string{"maxPriority": "5"},
+			}, func(_ context.Context, e *common.TopicEvent) (retry bool, err error) {
+				if err := sim(); err != nil {
+					log.Debugf("Simulated error - pubsub: %s, topic: %s, id: %s, data: %s", e.PubsubName, e.Topic, e.ID, e.Data)
+					return true, err
+				}
```

Comment:
I noticed that the certificate test failed, the core reason may be the `simulate` error introduced here.